src/index.ts

```diff
@@ -218,33 +218,9 @@ export function array <T>(element: Decoder<T>): Decoder<T[]> {
   })
 }
 
-/**
- * Decode an object with the given fields and types.
- * @param ad One or more EntryDecoders describing the object's fields.
- * @param cons Constructor that uses the results from the given EntryDecoders.
- * @returns A Decoder that will decode an object of the given type.
- */
-export function object <T, A>(ad: EntryDecoder<A>, cons: (a: A) => T): Decoder<T>
```

Comment:
I would like to see doc comments for `object` and `field`, as these are what is used to generate [the documentation](https://ooesili.github.io/type-safe-json-decoder/).